### ISSUE TITLE
Error "The property 'genericImageName' cannot be found on this object. Verify that the property exists" when re-running the script after it fails

### ISSUE THREAD

**Issue by AndriusAndrulevicius** [microsoft/navcontainerhelper#1094]
**Describe the issue**
If I run the script and it fails, when trying to re-run it I'm getting an error message "The property 'genericImageName' cannot be found on this object. Verify that the property exists"

**Scripts used to create container and cause the issue**
```
Remove-Variable * -ErrorAction SilentlyContinue
$containerName = 'democontainer'

$password = "P@ssword1" | ConvertTo-SecureString -asPlainText -Force
$username = "admin"
$credential = New-Object System.Management.Automation.PSCredential($username,$password)

$BCArtifactUrl = Get-BCArtifactUrl -type Sandbox -version 16.1.12629.12805 -country w1

New-BCContainer `
    -containerName $containerName `
    -accept_eula `
    -alwaysPull `
    -auth NavUserPassword `
    -Credential $credential `
    -artifactUrl $BCArtifactUrl `
    -shortcuts Desktop `
    -memoryLimit 8G `
    -updateHosts `
    -isolation hyperv `
    -EnableTaskScheduler:$false `
    -doNotExportObjectsToText `
    -doNotCheckHealth `
    -imagename democontainer:16.1.12629.12805
```
**Full output of scripts**
```
PS C:\Users\vmadmin> Remove-Variable * -ErrorAction SilentlyContinue
$containerName = 'democontainer'

$password = "P@ssword1" | ConvertTo-SecureString -asPlainText -Force
$username = "admin"
$credential = New-Object System.Management.Automation.PSCredential($username,$password)

$BCArtifactUrl = Get-BCArtifactUrl -type Sandbox -version 16.1.12629.12805 -country w1

New-BCContainer `
    -containerName $containerName `
    -accept_eula `
    -alwaysPull `
    -auth NavUserPassword `
    -Credential $credential `
    -artifactUrl $BCArtifactUrl `
    -shortcuts Desktop `
    -memoryLimit 8G `
    -updateHosts `
    -isolation hyperv `
    -EnableTaskScheduler:$false `
    -doNotExportObjectsToText `
    -doNotCheckHealth `
    -imagename democontainer:16.1.12629.12805
New-BCContainer : The property 'genericImageName' cannot be found on this object. Verify that the property exists.
At line:10 char:1
+ New-BCContainer `
+ ~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [New-NavContainer], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict,New-NavContainer
```

**Comment by freddydk** [comment]
Your remove-variable * line removes all configuration variables exported from NavContainerHelper.
I will see if I can make that more resilient towards this situation.

**Comment by freddydk** [comment]
I have changed the implementation to ensure that this doesn't happen.

**Comment by freddydk** [comment]
Shipped in 0.7.0.14
